geometry-1/DOMPoint-002.html

```diff
@@ -0,0 +1,171 @@
+            checkDOMPoint(new DOMPoint(1, 2, 3, 4, 5), {x:1, y:2, z:3, w:4});
+        },'test DOMPoint Constructor5');
+        test(function() {
+            checkDOMPoint(new DOMPoint(1, undefined), {x:1, y:0, z:0, w:1});
+        },'test DOMPoint Constructor6 undefined');
+        test(function() {
+            checkDOMPoint(new DOMPoint("a", "b"), {x:NaN, y:NaN, z:0, w:1});
+        },'test DOMPoint Constructor7 undefined');
+        test(function() {
+            checkDOMPoint(new DOMPoint({}), {x:NaN, y:0, z:0, w:1});
+        },'test DOMPoint Constructor8 Undefined');
+        test(function() {
+            checkDOMPoint(DOMPoint.fromPoint({}), {x:0, y:0, z:0, w:1});
+        },'test DOMPoint fromPoint0');
+        test(function() {
+            checkDOMPoint(DOMPoint.fromPoint({x:1}), {x:1, y:0, z:0, w:1});
+        },'test DOMPoint fromPoint1');
+        test(function() {
+            checkDOMPoint(DOMPoint.fromPoint({x:1, y:2}), {x:1, y:2, z:0, w:1});
+        },'test DOMPoint fromPoint2');
+        test(function() {
+            checkDOMPoint(DOMPoint.fromPoint({x:1, y:2, z:3}), {x:1, y:2, z:3, w:1});
+        },'test DOMPoint fromPoint3');
+        test(function() {
+            checkDOMPoint(DOMPoint.fromPoint({x:1, y:2, z:3, w:4}), {x:1, y:2, z:3, w:4});
+        },'test DOMPoint fromPoint4');
+        test(function() {
+            checkDOMPoint(DOMPoint.fromPoint({x:1, y:2, z:3, w:4, v:5}), {x:1, y:2, z:3, w:4});
+        },'test DOMPoint fromPoint5');
+        test(function() {
+            checkDOMPoint(DOMPoint.fromPoint({x:1, z:3}), {x:1, y:0, z:3, w:1});
+        },'test DOMPoint fromPoint6 irregular');
+        test(function() {
+            checkDOMPoint(DOMPoint.fromPoint({x:1, y: undefined, z:3}), {x:1, y:0, z:3, w:1});
+        },'test DOMPoint fromPoint7 undefined');
+        test(function() {
+            var point = new DOMPoint(5, 4);
+            var matrix = new DOMMatrix([2, 0, 0, 2, 10, 10]);
+            var result = point.matrixTransform(matrix);
+            var expected = getMatrixTransform(matrix, point);
+            checkDOMPoint(result, expected);
+        },'test DOMPoint matrixTransform');
+        test(function() {
+            var p = new DOMPoint(0, 0, 0, 1);
+            p.x = undefined;
+            p.y = undefined;
+            p.z = undefined;
+            p.w = undefined;
+            checkDOMPoint(p, {x:NaN, y:NaN, z:NaN, w:NaN});
+        },'test DOMPoint Attributes Undefined');
+        test(function() {
+            var p = new DOMPoint(0, 0, 0, 1);
+            p.x = NaN;
+            p.y = Number.POSITIVE_INFINITY;
+            p.z = Number.NEGATIVE_INFINITY;
+            p.w = Infinity;
+            checkDOMPoint(p, {x:NaN, y:Infinity, z:-Infinity, w:Infinity});
+        },'test DOMPoint Attributes NaN Infinity');
+        test(function() {
+            checkDOMPoint(new DOMPointReadOnly(), {x:0, y:0, z:0, w:1});
+        },'test DOMPointReadOnly Constructor0');
```

Comment:
Can you describe the test in the test name and avoid incrementing a number? e.g. `DOMPointReadOnly constructor no args`